lib/DDG/Spice/Symbolab.pm

```diff
@@ -102,19 +114,44 @@ handle query_lc => sub {
 			next;
 		}
 		
-		if (	$word !~ /\w\w\w\w/ and 
-                    ($word =~ /[\^\*\=\+\-\/\\]/ or 
-					 $word =~ /^\d*x$/)){
+		# i.e x^2, x^{2}
+		if ($word =~ /[xtu]\^(\{)?[xtu\d+](\})?/){
+			$minWeight -= 3;
+			next;
+		}
+		
+		# i.e 5i
+		if ($word =~ /[\d]i/){
+			$minWeight -= 3;
+			next;
+		}
+		
+		# i.e 'i'
+		if ($word eq 'i'){
+			$minWeight -= 1;
+			next;
+		}
+		
+		if ($word !~ /\w\w\w\w/ and ($word =~ /[\^\*\=\+\/\\\|]/ or $word =~ /^\d+x$/)){
```

Comment:
No need to escape inside `[]`:

I think this works: `if ($word !~ /\w{4}/ and ($word =~ /[*^=+/\|]/ or $word =~ /^\d+x$/)){`